### DISCUSSION TITLE
How to migrate software to reserve symbolic connections

### DISCUSSION THREAD

**Discussion by GunMoe** [ScoopInstaller/Scoop#5276] [Help and Questions]
![image](https://user-images.githubusercontent.com/39868439/205872732-ddcfc42c-bd90-4854-bc25-dd17eb9eff42.png)

The disk capacity is insufficient, and the disk needs to be switched

**Comment by HUMORCE** [comment]
you can specify paths of them in the config file.

or: 
```
rm <Scoop>\apps\nginx\current\logs -force
New-Item -ItemType Junction -Path <Scoop>\apps\nginx\current\logs  -Target D:\nginx\logs
```

**Comment by rashil2000** [comment]
The links in the screenshot are not symbolic links, but hard links (junctions). You can link specific folders as @HUMORCE mentioned, or change the Scoop home folder to some other drive.
